### ISSUE TITLE
On site load, all markers must be red

### ISSUE THREAD

**Issue by brentfraser** [nickfox/GpsTracker#49]
When map display site loads, it shows the last position of each session/route.  Each of these should be the same color (e.g. red) to indicate the last position.  The route display is ok; it shows the last position with the red marker and previous positions in green.

**Comment by nickfox** [comment]
Yes, that's by design to make it clear where the last known location is.

**Comment by brentfraser** [comment]
I found it confusing; I thought I was looking at a route.  I can see how it would be useful in the case of having many sessions/routes and only a couple of users.  I wonder if we could set it one way or the other based on an option in the maps.js?

**Comment by nickfox** [comment]
I slept on it and I can see your point of view. Go ahead if you would like to do that. (do the work on a branch if you could).
